### ISSUE TITLE
possbile asyncpg connection leak

### ISSUE THREAD

**Issue by maybe-hello-world** [netunicorn/netunicorn#95]
```
asyncpg.exceptions.TooManyConnectionsError: remaining connection slots are reserved for non-replication superuser connections
```

https://stackoverflow.com/questions/52542727/remaining-connection-slots-are-reserved-for-non-replication-superuser-connection

**Comment by maybe-hello-world** [comment]
Not reproducible, so let's just double-check all the session creation and disposal code

**Comment by maybe-hello-world** [comment]
Fixed in #122
